### PR TITLE
Install both mtda docker and qemu apt packages simultaneously 

### PR DESCRIPTION
**Pull request by bhargavdas**

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -23,11 +23,7 @@ override_dh_auto_install:
 	mv debian/mtda/usr/lib/python*/dist-packages/mtda/console/docker.py debian/mtda-docker/usr/lib/python3/dist-packages/mtda/console/
 	mv debian/mtda/usr/lib/python*/dist-packages/mtda/power/docker.py debian/mtda-docker/usr/lib/python3/dist-packages/mtda/power/
 	mv debian/mtda/usr/lib/python*/dist-packages/mtda/storage/docker.py debian/mtda-docker/usr/lib/python3/dist-packages/mtda/storage/
-	install -m 0755 -d debian/mtda-docker/etc/mtda
-	install -m 0644 configs/docker.ini debian/mtda-docker/etc/mtda/config
```
Comment (chombourger) [inline debian/rules:27]: ship to debian/mtda-docker/usr/share/doc/mtda-docker/examples/mtda.ini

File: debian/rules [hunk 2]
```diff
@@ -23,11 +23,7 @@ override_dh_auto_install:
...
 	:
-	install -m 0755 -d debian/mtda-kvm/etc/mtda
-	install -m 0644 configs/qemu.ini debian/mtda-kvm/etc/mtda/config
```
Comment (chombourger) [inline debian/rules:30]: ship to debian/mtda-docker/usr/share/doc/mtda-kvm/examples/mtda.ini

File: debian/rules (hunk 1 above)
Comment (bhargavdas) [reply debian/rules:27]: ok

File: docs/install.rst [hunk 3]
```diff
@@ -1,7 +1,7 @@
 Installation
 ============
 
-Using apt on Debian 11
+1.1 Using apt on Debian 11
```
Comment (chombourger) [inline docs/install.rst:4]: Numbering sections will cause issues in the Sphinx-generated documentation

File: docs/install.rst [hunk 4]
```diff
@@ -38,3 +38,20 @@ You may check your installation with the ``help`` command::
 
     $ export PATH=$HOME/.local/bin:$PATH
     $ mtda-cli help
+
+1.4 Using apt for installing mtda-docker and mtda-kvm
+-------------------------------------------------
+
+Package for both mtda-docker and mtda-kvm can be installed after executing either 1.1 or 1.2
```
Comment (chombourger) [inline docs/install.rst:45]: Docker and KVM may be used as virtual platforms by respectively installing the `mtda-docker` and `mtda-kvm` packages from the Apt package feeds described above.

File: docs/install.rst (hunk 3 above)
Comment (bhargavdas) [reply docs/install.rst]: removed

File: docs/install.rst (hunk 4 above)
Comment (bhargavdas) [reply docs/install.rst]: done

**Review by chombourger — APPROVED** [review]

**Pull request merged**